src/sheet_metal_conus.py

```diff
@@ -290,16 +423,21 @@ def beVerbose(self, dictCone, unitFactor, parent):
         simpletransform.applyTransformToNode(scale_matrix, text)
         text.text = "%4.2f" % (chord_base)
         if cut_dia >= 0.001:
+            centerx = ptA[0]*unitFactor + (ptD[0]-ptA[0])*unitFactor/2
```

Comment:
position the inner chord dimension near that line instead of straddling the outer chord line.